app/controllers/place_controller.rb

```diff
@@ -14,12 +12,15 @@ class PlaceController < ApplicationController
   REPORT_CHILD_ABUSE_SLUG = "report-child-abuse-to-local-council".freeze
 
   def show
+    set_content_item(PlacePresenter)
+
     if request.post?
       @location_error = location_error
       if @location_error
         @postcode = postcode
       elsif imminence_response.places_found?
-        @publication = PublicationWithPlacesPresenter.new(artefact, imminence_response.places)
+        @publication = PlacePresenter.new(content_item, imminence_response.places)
+        set_language_from_publication
```

Comment:
Nice, could probably just squash it into the commit where you made the change originally, unless you feel it requires its own commit? (sorry was in a meeting, looking through the rest of it now :) )